### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by stelfrag**
##### Summary
Some cleanup for queries and functions for readability

### CODE REVIEW

File: database/engine/rrdenginelib.h [hunk 1]
```diff
@@ -58,7 +58,7 @@ static inline void modify_bit(unsigned *x, unsigned pos, uint8_t val)
     }
 }
 
-#define RRDENG_PATH_MAX (4096)
+#define RRDENG_PATH_MAX (4096+1)
```
Comment (thiagoftsm) [inline database/engine/rrdenginelib.h:61]: Considering that we are talking about `PATH_MAX`, I suggest we use `FILENAME_MAX + 1`.

**Review by thiagoftsm — DISMISSED** [review]
PR is working as expected, LGTM!

**Review by MrZammler — APPROVED** [review]
I've checked the alert snapshots, after changing the chunk size to 10. Works ok!

**Review by MrZammler — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]
PR is working as expected, LGTM!

**Pull request merged**